.gitignore

```diff
@@ -22,6 +22,7 @@
 !/ext/financialacls
 !/ext/contributioncancelactions
 !/ext/recaptcha
+!/ext/oembed
```

Comment:
@totten it seems you have re-named the ext to be iframe but this is still set to be oembed?

Reply:
Thanks @seamuslee001. Fixed.